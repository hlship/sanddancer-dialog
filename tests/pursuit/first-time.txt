exit
(setup pursuit)
n
n
s
w
ne
n
n
