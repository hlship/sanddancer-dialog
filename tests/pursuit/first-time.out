

Middle of Nowhere (in the pickup truck)
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air. On the dash you
can see a piece of jade. The glove compartment is closed.

> exit
You work the rusty truck door open. You get out of the pickup truck.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (setup pursuit)
Pursuit should occur soon.
Query succeeded: (setup pursuit)
> n
You walk north.

Crumbling Concrete
This building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> n
You walk north.

Base of the Tower
Behind the concrete building a steel girder rises from the sand; far above you,
a red warning light blinks on and off. A tumbleweed drifts lazily against the
rusted metal among scrawny weeds bits of trash. Someone left a tin can here
years ago: ridged ripples run down its rusty side.

The desert stretches in all directions except back south towards the glow of the
building.

Something about the tumbleweed makes you want to brood about your last day of
high school.

> s
You walk south.

Crumbling Concrete
This building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

Out of the corner of your eye you spot a glimpse of something moving in the
darkness, off in the desert to the west.

> w
You head off in the direction of movement, scanning with your flashlight over
the skittering shadows.

Open Desert
Clouds of dust swirl through the air, and every inch of sand looks just like
every other.

Half-lost in the shadows, you see a looming cactus, some dry bones, and some
clumps of sagebrush and that's about it.

Hey, there it is again--something's moving to the northeast.

> ne
You chase after the shadow, faster and faster, breath misting in the cold air.

Open Desert
The desert stretches around you in all directions, and god damn it's dark.

All you can make out in the darkness are a dead cow, a low bluff, and a dull
grey boulder.

It's gone now... no, wait--a darting shadow to the north.

> n
Some crazy thrill comes over you. You feel like a hunter. You can almost smell
the thing you're chasing in the air. You can smell it, fur and terror. You race
after it.

Open Desert
The dust storm rages around you, and you shiver in the chill night air.

Half-lost in the shadows, you see a stand of cactus, a looming cactus, and some
dry bones and that's about it.

Hey, there it is again--something's moving to the north.

> n
Dodging after the fleeing, skittering shadow, you're almost caught up to it when
your foot trips on a rock and you stumble to the ground. But the ground doesn't
stop your fall, it collapses, and you're tumbling in a shower of dirt clods down
a hole, rolling down some crazy underground tunnel clutching at mud and roots,
till finally you slide into some damp, fetid place, and come up a stop against
something soft, huge, warm. And furry.

Burrow
You are here.

> 
